package.json

```diff
@@ -35,8 +37,8 @@
 		"jquery": "^3.5.1",
```

Comment:
Mentioning it just in case: This one still shows up, because it's still present in the source branches base. It will be gone after the merge, even though it does not look like it here.

Reply:
Rebased, just to be clear.